core-bundle/src/Resources/contao/classes/DataContainer.php

```diff
@@ -1764,7 +1764,7 @@ public function generateRecordLabel(array $row, string $table = null, bool $prot
 
 		// Remove empty brackets (), [], {}, <> and empty tags from the label
 		$label = preg_replace('/\( *\) ?|\[ *] ?|{ *} ?|< *> ?/', '', $label);
-		$label = preg_replace('/<[^>]+>\s*<\/[^>]+>/', '', $label);
+		$label = preg_replace('/<[^\/!][^>]+>\s*<\/[^>]+>/', '', $label);
```

Comment:
Could we instead remove this line completely? Or is it important that empty tags get removed here?

Reply:
If you have a label such as `%s <span class="foo">(%s)</span>` and the second `%s` is empty, it will now be converted to `Label `. Without the line, it would be converted to `Label <span class="foo"></span>`.

Reply:
I’d be fine with `Label <span class="foo"></span>` ☺️

Reply:
Hm… 🤔

Reply:
As discussed in the call, we want to try removing the line. If it does not work, we will merge the PR.

Reply:
As discussed in the call, we want to try removing the line. If it does not work, we will merge the PR.

Reply:
Ok, I think we will merge the PR.

### Before

<img width="719" alt="" src="https://user-images.githubusercontent.com/1192057/236849977-8c18a6eb-b669-4b6b-a32e-af1e8776d505.png">

### After

<img width="630" alt="" src="https://user-images.githubusercontent.com/1192057/236850019-80410288-8689-4c94-b606-c37a487b2e3a.png">

The empty `<span>` creates a visible spacing between the first and the third value.